app/components/graph-editor/README.md

```diff
@@ -0,0 +1,82 @@
+# Notes
```

Comment:
I like this idea. I also stole the idea of making a readme from Sheyne, so credit where credit is due. It made sense to make a readme, since the header comment in `graph-editor.component` was getting pretty long.